packages/components/src/FormField/FormField.css

```diff
@@ -130,6 +130,10 @@
   width: auto;
 }
 
+.inline .select select {
+  padding-right: calc(var(--field--padding-right) + var(--base-unit) * 2);
+}
+
```

Comment:
Does it look off because the inline selects in these examples use the `small` size? (I feel like that might be the case because we don't adjust the left or right padding on inputs when their size increases, so it's currently optimized for `base` size), or is it something specific to the inline select?

Reply:
That might be _part_ of the issue. But from what I can tell, this specifically impacts `inline` selects that are shrinking to their min content.

When the browser paints the select, it sizes it so it can fit it's largest `<option>` but it obviously isn't painting space for the caret because it's absolutely positioned so it's been pulled from flow (and thus not part of the size calc during the paint).

Reply:
OK, that's (maybe?) even simpler. If we _can_ bring the caret back to a more snug fit with the border, let's do it!

Reply:
I also don't know if that's entirely coupled to the initial problem you're solving for, if it isn't we can solve for this problem in a follow-up.

Reply:
My initial problem just became less important because I'm solving it by using Combobox instead 🤣